sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java

```diff
@@ -188,6 +189,33 @@ interface Blank extends DefinitionWithRegion<WithGroup> {
         interface WithGroup extends GroupableResource.DefinitionStages.WithGroup<WithVersion> {
         }
 
+        /**
+         * The stage of the Kubernetes cluster definition allowing to specify AKS pricing tier.
+         */
+        interface WithPricingTier {
+
+            /**
+             * Specifies the pricing tier  the Kubernetes cluster is free.
```

Comment:
nit, double spaces between `tier` and `the`

Reply:
Fixed in the new version.